Core/CMP API Specification.md

```diff
@@ -458,8 +459,31 @@ For example, client can ask the CMP to get the IAB TCF CA v1.0 TCData:
 
 
 ```javascript
-__gpp('getSection', myFunction, "tcfcav1");
+__gpp('getSection', myFunction, "tcfca"); 
```

Comment:
pingReturn gives a list of applicable section numbers. How is an api caller supposed to know which section string to call for some number? Should they only listen for sectionChange?
![image](https://user-images.githubusercontent.com/1683175/234375984-b3f8604e-8e0f-4700-978e-94e7bc068a54.png)